src/cljs/main/broadfcui/page/method_repo/methods_configs_acl.cljs

```diff
@@ -111,8 +113,11 @@
               :payload non-empty-acls-w-public
               :on-done (net/handle-ajax-response
                          (fn [{:keys [success? parsed-response] :as response}]
-                           (swap! state assoc :acl-response response)
-                           (swap! state dissoc :saving?)))})))))
+                           (if success?
+                             (modal/pop-modal)
+                             (do
+                               (swap! state assoc :persist-error (:message parsed-response))
+                               (swap! state dissoc :saving?)))))})))))
```

Comment:
Can we verify this? I've read that React will batch state changes. If it does indeed update on each one, I want to be checking for this in my own reviews.